src/scripts/setup.sh

```diff
@@ -10,7 +10,7 @@ apt update -y
 apt install git nodejs openjdk-11-jre-headless -y
 apt install npm -y
 
-sudo -u mcuser -i << EOF
```

Comment:
Still need the EOF part

Reply:
`EOF` stands for End of file, right? So shouldn't it go at the end of the script?

Reply:
Sorry it's confusing. Yes it does, but this is actually a Bash scripting thing. It's saying run the following commands until the `EOF` within the context of this user mcuser rather than root. So everything between this line and the plain `EOF` at the end of the script is run as that user.